bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_TTH_TTToAll_HToAATo4B_M-MASS/SUSY_TTH_TTToAll_HToAATo4B_M-MASS_run_card.dat

```diff
@@ -0,0 +1,270 @@
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ lhapdf       = pdlabel     ! PDF set                                     
+ 325300       = lhaid
```

Comment:
Is there a reason not to use $DEFAULT_PDF_SETS?

Reply:
Hi @sihyunjeon ,

According to slide 3 here, the Higgs group recommendation for UL is NNPDF31_nnlo_as_0118_mc_hessian_pdfas
https://indico.cern.ch/event/996925/contributions/4188686/attachments/2195237/3711418
/PAGNews_23022021.pdf

This PDF set appears in this list as well, with ID = 325300:
https://lhapdf.hepforge.org/pdfsets

Based on the following twiki, I concluded that I should then set lhaid = 325300:
https://twiki.cern.ch/twiki/bin/viewauth/CMS/QuickGuideMadGraph5aMCatNLO#PDF_Choice_for_2017_production

If you would prefer that I set it to $DEFAULT_PDF_SETS, that's fine.  I just wasn't sure how to make sure that $DEFAULT_PDF_SETS = 325300.

Thanks,
Andrew

Reply:
$DEFAULT_PDF_SETS will be read from this [1]. So it does include 325300

[1] https://github.com/cms-sw/genproductions/blob/master/MetaData/pdflist_5f_2017.dat#L2

Reply:
Changed to $DEFAULT_PDF_SETS